scylla-gdb.py

```diff
@@ -1295,6 +1296,11 @@ def _make_symbol_matchers(symbol_specs):
     def _make_symbol_matcher(symbol_spec):
         unmangled_prefix = 'vtable for {}'.format('::'.join(symbol_spec))
         def matches_symbol(name):
+            name = name.strip()
+            if name.endswith(")") or (") [clone ." in name and name.endswith("]")):
```

Comment:
`endswith(")")` indeed sounds rather generic; did you test manually how many false positives are actually found when calling `scylla task_histogram` or any other function that uses this code?

Reply:
tried it on the nice huge coredump from #10933, didn't see any false positives actually, but who knows